src/OpenLoco/GameState.h

```diff
@@ -8,6 +8,7 @@
 #include "Message.h"
 #include "Station.h"
 #include "Town.h"
+#include <OpenLoco/Scenario.h>
```

Comment:
use "" instead of <>

Reply:
Bloody visual studio just not helping with the automatic includes :(